src/WarcraftLegacies.Source/Setup/Legends/LegendNeutral.cs

```diff
@@ -26,7 +26,7 @@ public static class LegendNeutral
     public static LegendaryHero Vaelastrasz { get; private set; }
     public static Capital Caerdarrow { get; private set; }
     public static Capital Shaladrassil { get; private set; }
-    
+    public static Capital ThandolSpanDestroyed { get; private set; }
```

Comment:
I wonder if there is a good reason to use a Capital for this? You only need the destructable aspect right, so it could just be a plain destructable?

Reply:
I need a unit to be able to use objectives. Or can I convert that somehow?

Reply:
I think you should be able to just use the `ObjectiveControlUnit` objective instead, and use the `PreplacedUnitSystem` to pass in the appropriate unit as a parameter. You would then need to make the unit capturable separately, which you can do with the `MakeCapturable` extension method.

Reply:
Sorry again, I failed to read properly. I see that the objective you're using is actually `ObjectiveLegendReachesFullHealth`. I think we should make a new `ObjectiveUnitReachesFullHealth`, but that could be done after this PR is merged if you'd prefer.

Reply:
Yea my bad, I don't need a capital anymore. I used a different objective at first which required a capital but not anymore so I'll changed that